ntbs-service/Pages/ValidationModel.cs

```
@@ -43,7 +43,10 @@ private void SetProperty(object model, string key, object value)
             {
                 value = converter.ConvertFrom(value);
             }
-            catch {}
+            catch 
+            {
+                // Value is a type of object and Complex objects may results in error when casting it to a necessary type

```

Fair, but this doesn't answer why is it OK for us to ignore the error.